src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Codeunit.al

```diff
@@ -5,21 +5,163 @@
 
 namespace System.TestTools.AITestToolkit;
 
+using System.TestTools.TestRunner;
+
 codeunit 149032 "AIT Log Entry"
 {
     Access = Internal;
 
+    var
+        LineNoFilterLbl: Label 'Codeunit %1 "%2" (Input: %3)', Locked = true;
+
     procedure DrillDownFailedAITLogEntries(AITSuiteCode: Code[100]; LineNo: Integer; VersionNo: Integer)
     var
         AITLogEntries: Record "AIT Log Entry";
+    begin
+        AITLogEntries.SetRange(Version, VersionNo);
+        DrillDownFailedAITLogEntries(AITLogEntries, AITSuiteCode, LineNo);
+    end;
+
+    procedure DrillDownFailedAITLogEntries(AITSuiteCode: Code[100]; LineNo: Integer; Tag: Text[20])
+    var
+        AITLogEntries: Record "AIT Log Entry";
+    begin
+        AITLogEntries.SetRange(Tag, Tag);
+        DrillDownFailedAITLogEntries(AITLogEntries, AITSuiteCode, LineNo);
+    end;
```

Comment:
Why not just set filter where you call this function? instead of having two new functions that does nothing

Reply:
They don't do nothing. I think it's nice having this in it's own separate function and not having to do part of the filtering happen on the page and the other part in the codeunit. This makes it clearer what's happening and what's being filtered on imo.